tests/integration_tests/functional/test_metrics.py

```diff
@@ -23,4 +26,41 @@ def test_flush_metrics(test_microvm_with_api):
 
     microvm.start()
 
+    cmd = 'read line < {} && echo $line'.format(metrics_fifo_path)
+    res = utils.run_cmd(cmd)
```

Comment:
Can we use `logging.py::Fifo::sequential_reader()` here ? Or `metrics_fifo_path.readlines()` ? Or `metrics_fifo_path.read()` ?

Reply:
Done.